python/cudf/cudf/tests/test_parquet.py

```diff
@@ -281,21 +282,54 @@ def num_row_groups(rows, group_size):
 
 
 @pytest.mark.parametrize("row_group_size", [1, 5, 100])
-def test_parquet_read_row_group(tmpdir, pdf, row_group_size):
+def test_parquet_read_row_groups(tmpdir, pdf, row_group_size):
+    if "col_category" in pdf.columns:
+        pdf = pdf.drop(columns=["col_category"])
     fname = tmpdir.join("row_group.parquet")
     pdf.to_parquet(fname, compression="gzip", row_group_size=row_group_size)
 
     num_rows, row_groups, col_names = cudf.io.read_parquet_metadata(fname)
 
-    gdf = [cudf.read_parquet(fname, row_group=i) for i in range(row_groups)]
-    gdf = cudf.concat(gdf).reset_index(drop=True)
+    gdf = [
+        cudf.read_parquet(fname, row_group_list=[i]) for i in range(row_groups)
```

Comment:
Can we keep `i` as a scalar for the row group?

Reply:
done